app/models/measure_test.rb

```diff
@@ -85,4 +85,8 @@ def cat3_status
       'unstarted'
     end
   end
+
+  def provider
+    Provider.find(records.first.provider_performances.first.provider_id)
```

Comment:
Can we directly assign the provider to the model, instead of having to trainwreck this?